### ISSUE TITLE
Wet days - using threshold of 0 not 1 -replace custom aggregator with count > 1

### ISSUE THREAD

**Issue by nhsavage** [MetOffice/PyPRECIS#140]
in the core EAS notebooks the example for how to calculate the number of wet days (worksheet 5, section 5.1) is wrong. 

the aggregator used is:

```
count_nonzero = Aggregator('count', None,
                           units_func=lambda units: 1,
                           lazy_func=da.count_nonzero)
```

this counts the number of days when the rainfall is not zero. The correct definition is precipitation >=1 mm/day

the correct approach is shown on the AFR-22 version:

https://github.com/MetOffice/PyPRECIS/blob/new/afr_22/notebooks/AFR-22/worksheet5.ipynb 

```
        # count wet days 
        model_wetdays = data.collapsed('time', iris.analysis.COUNT,
                               function=lambda values: values > 1)
```

**Comment by rosannaamato** [comment]
Updated code in https://github.com/MetOffice/PyPRECIS/blob/3defd940f0c4025f79adc402e5b83b9ba6914e84/notebooks/worksheet5.ipynb with AFR22 version of wetday_count in ab0d48b / 72fed13. 
Quick test of plot shows sensible looking output: 
<img width="508" alt="worksheet5_wetdays" src="https://user-images.githubusercontent.com/34257276/183086898-9ca72bb6-874f-4f41-b96e-75e0ad0d9c4f.png">

**Comment by rosannaamato** [comment]
Wet day count also needed updating in 5.1b of  `notebooks/worksheet_solutions.ipynb`
5.1e solutions was missing a line for `pcent_bias_subset`. Added. 
847328d

**Comment by nhsavage** [comment]
closed by #143
